### PR TITLE
Write and clear functionality

### PR DESCRIPTION
**Pull request by dcadenas**
Solves  #1

This let's us do

```ruby
sr = SheetReaderWriter.new('1Q2NdvsSECbDrdOf9-C1EhzHq__3jWW3lQWDDB0mJbd8')

#updates the whole sheet
sr.write  [
   ["foo",   "bar"],
   ["hey",    "ho"],
   ["let's",   nil],
   [nil,      "go"]
]

#clears the whole sheet
sr.clear
```

### CODE REVIEW

**Comment by dcadenas** [comment]
@harlow exactly, and the first row is important as those map to the keys that will be returned by the read method

EDIT:
@harlow scratch that, while using this in the private repo I'm working on I noticed it is much easier to just implement the same format returned by the read method for the writes so we don't need to deal with the column names case and empty values and it's mentally easier. So now it's
```ruby
screen_reader_writer.write [
  {"foo"=>"hey", "bar"=>"ho"},
  {"foo"=>"let's"},
  {"bar"=>"go"}
]
```

File: README.md [hunk 1]
```diff
@@ -37,25 +44,46 @@ This would be mapped to a spreadsheet with this structure:
...
+sheet_reader_writer.read("Some other sheet")
+```
+
+### Write
+
+Substitutes the content of the spreadhseet with the specified values.
+The format is the same as the one returned by the read method.
+
+```ruby
+screen_reader_writer.write [
```
Comment (harlow) [inline README.md:59]: this is great 👍

Comment (harlow) [reply README.md:59]: if a col doesn't exist in sheets will it create it?

File: test/test_sheet_reader_writer.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      result = SheetReaderWriter.new('1Q732RcSCYJpWZZtnKx4yXVQTgjSEbilJKznsOp-qlPE').read
+    end
+  end
+
+  def test_with_sample_sheet
+    # Be sure to run the test with the correct env variables, this is an integration test that requires internet
+    sr = SheetReaderWriter.new('1bDmIkemWGk2wQXhzyAbtraIxMEETjn7XGgQjbna7SIY')
+
+    sr.clear
+
```
Comment (harlow) [inline test/test_sheet_reader_writer.rb:32]: nice end-to-end spec 👍

**Review by harlow — APPROVED** [review]
Looks great. Lets 🚢

**Pull request merged**